dcsazure_adls_to_adls_prof_pl/README.md

```diff
@@ -81,52 +97,42 @@ The default value of `P_SUFFIX_DELIMITER_MAP` can be more easily read when we ap
 {
   "csv": {
     "column_delimiter": ",",
-    "row_delimiter": "\\r\\n",
     "quote_character": "\"",
-    "escape_character": "\\\\",
-    "first_row_as_header": true,
+    "escape_character": "\\",
     "null_value": ""
   },
   "txt": {
     "column_delimiter": "|",
-    "row_delimiter": "\\r\\n",
     "quote_character": "\"",
-    "escape_character": "\\\\",
-    "first_row_as_header": true,
+    "escape_character": "\\",
     "null_value": ""
   },
   "NO_EXT": {
     "column_delimiter": "|",
-    "row_delimiter": "\\r\\n",
     "quote_character": "\"",
-    "escape_character": "\\\\",
-    "first_row_as_header": true,
+    "escape_character": "\\",
     "null_value": ""
   }
 }
 ```
 As the pipeline runs, it will refer to these values when parsing the files in ADLS. If a configuration for the row or
 column delimiter is incorrect, the profiling may or may not fail. If the column delimiter was incorrect, then the
 results in the `discovered_ruleset` table will be incorrect. For example, if the ADLS store has a `csv` file whose
-header is: `column1,column2,column3`, and the column delimiter for `csv` is specified as `|`, then there will be an
+header is: `column1|column2|column3`, and the column delimiter for `csv` is specified as `,`, then there will be an
 entry in the `discovered_ruleset` table corresponding to table name `csv` with `identified_column` containing
-`column1,column2,column3`. To correct this, you will need to remove the erroneous row from the table, and re-run the
-pipeline with the correct column delimiter.
+`column1|column2|column3`. To correct this, you will need to remove the erroneous row from the table, and re-run the
+pipeline with the correct column delimiter. If you have a mix of delimiters in files of the same extension across
+subdirectories, you can either 
```

Comment:
Is this sentence unfinished? Maybe its just the diff view.

Reply:
Good catch, fixed it.